metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/indexing_master.py

```diff
@@ -115,6 +116,16 @@ def elasticsearch_template_delete(self, env):
         yaf_cmd = ambari_format('curl -s -XDELETE "http://{es_http_url}/yaf_index*"')
         Execute(yaf_cmd, logoutput=True)
 
+    def zeppelin_dashboard_install(self, env):
+        from params import params
+        env.set_params(params)
+
+        for dirName, subdirList, files in os.walk(params.metron_config_zeppelin_path):
```

Comment:
Would it make sense to log the path that it is looking for the Notebooks in; `params.metron_config_zeppelin_path`?  I'd like to see this in the Ambari status output, especially if it doesn't find any Notebooks.